include/mbedtls/ssl.h

```diff
@@ -1783,6 +1801,10 @@ struct mbedtls_ssl_context
                             *   and #MBEDTLS_SSL_CID_DISABLED. */
 #endif /* MBEDTLS_SSL_DTLS_CONNECTION_ID */
 
+#if defined(MBEDTLS_SSL_EARLY_DATA) && defined(MBEDTLS_SSL_CLI_C)
+    int MBEDTLS_PRIVATE(early_data_status);
```

Comment:
Add a brief description.

Reply:
This description is removed cause it has no need, see comment https://github.com/Mbed-TLS/mbedtls/pull/6486#discussion_r1015174308